src/Entities/Payout.php

```diff
@@ -0,0 +1,586 @@
+<?php
+/**
+ * This source file is proprietary and part of Rebilly.
+ *
+ * (c) Rebilly SRL
+ *     Rebilly Ltd.
+ *     Rebilly Inc.
+ *
+ * @see https://www.rebilly.com
+ */
+
+namespace Rebilly\Entities;
+
+use Rebilly\Rest\Entity;
+
+/**
+ * Class Payout.
+ */
+final class Payout extends Entity
```

Comment:
This object contains getters for unknown properties. I guess they added because this object is projected as possible response, which is not so. According to API definition the server responds with `Transaction` object on payout operation.

Reply:
Correct. I removed them